### PR TITLE
better docs for FTL

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

**Comment by aaron-prindle** [comment]
fixes #540

File: ftl/README.md [hunk 1]
```diff
@@ -20,6 +20,27 @@ This command can be read as "Build the source code in directory `$dir` into an i
 
 These binaries **do not depend on Docker**, and construct images directly in the registry.
 
+So for example will we demonstrate using the Node FTL builder to create a container for a node application from a node app's source code:
```
Comment (nkubala) [inline ftl/README.md]: s/'So for example'/'As an example'

File: ftl/README.md [hunk 2]
```diff
@@ -20,6 +20,27 @@ This command can be read as "Build the source code in directory `$dir` into an i
...
+Assume we are deploying the node source for the app https://github.com/JustinBeckwith/cloudcats.  First downlowad the node ftl.par file from https://storage.googleapis.com/gcp-container-tools/ftl/node/latest/ftl.par.  Then we run the ftl.par file pointing to our application:
```
Comment (nkubala) [inline ftl/README.md]: downlowad -> download

**Comment by aaron-prindle** [comment]
Working on changing the documents to be markdown

File: ftl/README.md [hunk 3]
```diff
@@ -20,6 +20,27 @@ This command can be read as "Build the source code in directory `$dir` into an i
...
+```shell
+$ ftl.par --directory=$HOME/cloudcats/web --base=gcr.io/google-appengine/nodejs:latest --image=gcr.i
+o/my-project/cloudcats-node-app:latest
+```
+
+## Releases
+Currently FTL is released in .par format for each supported runtime.
+node:
+[https://storage.googleapis.com/gcp-container-tools/ftl/node/latest/ftl.par]()
```
Comment (nkubala) [inline ftl/README.md]: Looks like you meant to do something like 

```
[FTL node par](https://storage.googleapis.com/gcp-container-tools/ftl/node/latest/ftl.par)
```
for these links.

File: ftl/README.md (hunk 1 above)
Comment (aaron-prindle) [reply ftl/README.md]: done

File: ftl/README.md (hunk 2 above)
Comment (aaron-prindle) [reply ftl/README.md]: done

File: ftl/README.md (hunk 3 above)
Comment (aaron-prindle) [reply ftl/README.md]: changed the format here

File: ftl/README.md [hunk 4]
```diff
@@ -20,6 +20,27 @@ This command can be read as "Build the source code in directory `$dir` into an i
 
 These binaries **do not depend on Docker**, and construct images directly in the registry.
 
+As an example, will we demonstrate using the Node FTL builder to create a container for a node application from a node app's source code:
```
Comment (nkubala) [inline ftl/README.md]: s/'will we'/'we will'

File: ftl/README.md [hunk 5]
```diff
@@ -28,3 +49,7 @@ python ftl/ftl_node_integration_tests_yaml.py | gcloud container builds submit -
 python ftl/ftl_php_integration_tests_yaml.py | gcloud container builds submit --config /dev/fd/0 .
 gcloud container builds submit --config ftl/ftl_python_integration_tests.yaml .
 ```
+
+## FTL Runtime Design Documents
+[php](https://docs.google.com/document/d/1AB255g8N-J7IYEbhmiTRf29Ox1afEgs3df8GK4NBhrk/edit?usp=sharing)
```
Comment (dlorenc) [inline ftl/README.md]: These links probably won't work publicly. Consider leaving them out until you port them to markdown.

Comment (aaron-prindle) [reply ftl/README.md]: I made this links to work publicly from my other gmail account.  These documents are community docs w/ internal Google references removed

**Pull request merged**